### PR TITLE
Add isStandardFocusableMode private globalConfig 

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Spotlight needs to navigate standard focusable element when isStandardFocusableMode is set true.
If focusRingElement exists, when the spotlight focuses a certain element, focusRingElement needs to move.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Implement spotlight logic to determine if the element is standard focusable element.
Add `isStandardFocusMode` config in GlobalConfig(https://github.com/enactjs/enact/blob/master/packages/spotlight/src/container.js#L43).
`isStandardFocusMode` config can be set with spotlight.initialize function.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
isStandardFocusMode config is the private config.

### Links
[//]: # (Related issues, references)
WRP-10235

### CODE REVIEW

File: packages/spotlight/src/container.js [hunk 1]
```diff
@@ -89,7 +90,10 @@ let GlobalConfig = {
  * @private
  */
 const querySelector = (node, includeSelector, excludeSelector) => {
-	const include = new Set(node.querySelectorAll(includeSelector));
+	let include = new Set(node.querySelectorAll(includeSelector));
```
Comment (0x64) [inline packages/spotlight/src/container.js:93]: Let's move `querySelectorAll` call into two lines below as we a new `Set` here is not used actually.

File: packages/spotlight/src/utils.js [hunk 2]
```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
 	return getRect(containerNode);
 }
 
+function isStandardFocusable (element) {
+	if ((element.tabIndex < 0) || isAtagWithoutHref(element) || isActuallyDisabled(element) || isExpresslyInert(element) || isElementHidden(element)) {
```
Comment (0x64) [inline packages/spotlight/src/utils.js:176]: For later visits, could you please add some comments for clauses in this function?

File: packages/spotlight/src/utils.js [hunk 3]
```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
...
+		return false;
+	} else if ((!element.parentElement) || (element.tabIndex >= 0)) {
+		return true;
+	}
+}
```
Comment (0x64) [inline packages/spotlight/src/utils.js:181]: Please add `return false;` explicitly.

File: packages/spotlight/src/utils.js [hunk 4]
```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
 	return getRect(containerNode);
 }
 
+function isStandardFocusable (element) {
+	if ((element.tabIndex < 0) || isAtagWithoutHref(element) || isActuallyDisabled(element) || isExpresslyInert(element) || isElementHidden(element)) {
+		return false;
+	} else if ((!element.parentElement) || (element.tabIndex >= 0)) {
+		return true;
+	}
```
Comment (0x64) [inline packages/spotlight/src/utils.js:180]: Please add `else` case to return falsy value for rest conditions.

File: packages/spotlight/src/utils.js [hunk 5]
```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
...
+}
+
+function isAtagWithoutHref (element) {
+	return (element.tagName === 'A' && element.getAttribute('href') === null && element.getAttribute('tabIndex') === null);
+}
+
+function isActuallyDisabled (element) {
+	if (['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA', 'OPTGROUP', 'OPTION', 'FIELDSET'].includes(element.tagName)) {
+		return (element.disabled);
```
Comment (0x64) [inline packages/spotlight/src/utils.js:189]: Not an issue but just curious that why do you use parenthesis here? :>

File: packages/spotlight/src/utils.js [hunk 6]
```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
...
+		return false;
+	}
+}
+
+function isExpresslyInert (element) {
+	return ((element.inert) && (!element.ownerDocument.documentElement.inert));
+}
+
+function isElementHidden (element) {
+	let elemRect = element.getBoundingClientRect();
```
Comment (0x64) [inline packages/spotlight/src/utils.js:200]:
```suggestion
	const elemRect = element.getBoundingClientRect();
```

File: packages/spotlight/src/utils.js (hunk 5 above)
Comment (juwonjeong) [reply packages/spotlight/src/utils.js]: I think it is unnecessary. The related code has been updated.

File: packages/spotlight/src/utils.js (hunk 4 above)
Comment (juwonjeong) [reply packages/spotlight/src/utils.js]: I added else case. Thank you.

File: packages/spotlight/src/utils.js (hunk 2 above)
Comment (juwonjeong) [reply packages/spotlight/src/utils.js]: I added some comments

File: packages/spotlight/src/container.js (hunk 1 above)
Comment (juwonjeong) [reply packages/spotlight/src/container.js]: It's fixed.

**Review by 0x64 — APPROVED** [review]

**Pull request merged**